flink-runtime/src/main/java/org/apache/flink/runtime/rest/versioning/RestAPIVersion.java

```diff
@@ -31,7 +31,7 @@
  * modification of a handler url - addition of new mandatory parameters - removal of a
  * handler/request - modifications to request/response bodies (excluding additions)
  */
-public enum RestAPIVersion {
+public enum RestAPIVersion implements APIVersion {
```

Comment:
```suggestion
public enum RuntimeRestAPIVersion implements APIVersion {
```

Reply:
I think we should also find a way to define `getLatestVersion` and the comparing method in the common abstract. The unused `fromURLVersionPrefix` probably can be removed.